_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -397,7 +397,7 @@ mfxStatus SetRateControl(
         rate_param->ICQ_quality_factor = par.mfx.ICQQuality;
 #ifdef MFX_ENABLE_QVBR
     else if (par.mfx.RateControlMethod == MFX_RATECONTROL_QVBR)
-        rate_param->quality_factor = extOpt3.QVBRQuality;
+        rate_param->ICQ_quality_factor = extOpt3.QVBRQuality;
```

Comment:
hmmm. Why not quality_factor? There is a note for the ICQ_quality_factor: "this value will be deprecated in future, to use quality_factor instead of it." Do we need a follow up with the driver team on that?

Reply:
It looks like we promoted this change ahead of driver. There are https://github.com/intel/media-driver/pull/632 which replace `ICQ_quality_factor` with `quality_factor`.